config.toml

```diff
@@ -131,25 +131,31 @@ description = "Production-Grade Container Orchestration"
 languageName ="English"
 # Weight used for sorting.
 weight = 1
+
 [languages.cn]
 title = "Kubernetes"
 description = "Production-Grade Container Orchestration"
 languageName = "Chinese"
 weight = 2
 contentDir = "content/cn"
+
+[languages.ko]
+title = "Kubernetes"
+description = "Production-Grade Container Orchestration"
+# languageName = "Korean"
+languageName = "한국인"
```

Comment:
@gochist @ClaudiaJKang I used Google here, please correct this if it's wrong or if there's a better idiom!